aten/src/ATen/native/cuda/ForeachFunctors.cuh

```diff
@@ -7,6 +7,16 @@ namespace at { namespace native {
 
 namespace {
 
+// TODO(crcrpar): Handle version bump in codegen.
+// rel: https://github.com/pytorch/pytorch/blob/9cf84347767c8abb8feba18a9a1baba321eeb8b9/tools/autograd/gen_inplace_or_view_type.py#L481-L482
+inline void maybe_increment_version(TensorList tensors) {
+  for (const auto & t : tensors) {
+    if (at::GradMode::is_enabled()) {
```

Comment:
Ho sorry my comment wasn't clear enough.
The whole if was not ok. This is not file either. Inplace changes in no_grad mode must also be tracked.
In short, you must always bump the version, whatever the state of autograd.